omnibus/config/projects/agent.rb

```diff
@@ -336,6 +347,9 @@
     # Check the exported symbols from the binary
     inspect_binary(bin, &raise_if_forbidden_symbol_found)
 
+    # Check that CNG symbols are present
+    inspect_binary(bin, &raise_if_fips_symbol_not_found)
```

Comment:
Similar remark here, if --fips-mode isn't provided to omnibus.build we don't want to ensure the presence of FIPS symbols